library/src/main/kotlin/com/oddin/oddsfeedsdk/config/OddsFeedConfiguration.kt

```diff
@@ -34,11 +33,20 @@ class OddsFeedConfigurationBuilder internal constructor() {
     private var initialSnapshotRecoveryInterval: Duration? = null
 
     fun selectProduction() = apply {
```

Comment:
I suggest to rename to selectProductionEnvironment(). It would be easier to understand that you suppose this method or selectEnvironment() method.